src/js/helpers/mapController/layerSorting.ts

```diff
@@ -0,0 +1,49 @@
+import Map from 'esri/Map';
+import { sortBy, flatten } from 'lodash-es';
```

Comment:
@vaidotasp looks like `@types/lodash-es` could help type `lodash-es`, otherwise we'll get the compile error `Could not find a declaration file for module 'lodash-es'`